rye/src/pyproject.rs

```diff
@@ -159,7 +159,7 @@ impl SourceRef {
             .map(|x| x.to_string())
             .ok_or_else(|| anyhow!("expected source.url"))?;
         let verify_ssl = source
-            .get("verify_ssl")
```

Comment:
Should we read both `verify_ssl` and `verify-ssl` for backward compatibility?

Reply:
Thanks for pointing that out. I have fixed it.